gauntlet/packages/gauntlet-solana-contracts/src/commands/contracts/ocr2/proposePayees.ts

```diff
@@ -42,6 +43,7 @@ export default class SetPayees extends SolanaCommand {
     super(flags, args)
 
     this.requireFlag('state', 'Provide a valid state address')
+    this.requireFlag('proposalId', 'Provide a valid proposal address')
```

Comment:
what's the different between `state` and `proposalId`?

It doesn't look like `state` is being used.

Reply:
`state` is used here: https://github.com/smartcontractkit/chainlink-solana/blob/f0ee38f2779dcc29dc2b5a4e5caf238b5e422cfd/gauntlet/packages/gauntlet-solana-contracts/src/commands/contracts/ocr2/proposePayees.ts#L30

`state` corresponds to configs read from the RDD, and `proposalID` corresponds to account where the new offchain config is temporarily store before it is finalized/approved